_pages/en_US/404.txt

```diff
@@ -5,5 +5,5 @@ excerpt: "Page not found. Your pixels are in another canvas." #
 sitemap: false
 ---
 
-Sorry, but the page you were trying to view does not exist.
+Sorry, but the page you were trying to view does not exist. This may be because the page was removed due to being outdated. 
```

Comment:
A bit oddly worded, and would like to link people to get started or the homepage.

A problem with just wording it as a maybe is that discord still drops closing brackets unless you append # or .html, meaning we could have any current link also 404, which would confuse people into thinking whatever current method is outdated.